### DISCUSSION TITLE
All my Lora's disappeared on Swarm.

### DISCUSSION THREAD

**Discussion by rhd2330** [mcmonkeyprojects/SwarmUI#265] [Q&A]
All my Lora's disappeared on Swarm.

All my Lora's disappeared, and the dropdown settings below it had disappeared. I don't know what I did; I was messing around with the depth settings, and everything disappeared. I have tried to get it back, but nothing I have done is working. This is the error it is showing me:

Error: Invalid value '' for parameter 'depth', must be type 'Int32'

I have not gotten it fixed yet, but if you would like to try to do what I think I did, I wouldn't advise it, but maybe you want to try. Everything else works fine; it's just everything under the Lora's disappears, and I mean everything.

In the depth field, type 1.1; that is what I think I did, and everything on Swarm vanished under Loras.

You may say, "Just type in a different value," but the place where you type in a different value is gone. There is nothing there. Everything else is working. If there is another location to get to it, please let me know. Thank you.

**Comment by mcmonkey4eva** [comment]
First of all: are you sure you're running SwarmUI v0.9.2.2? as opposed to the legacy StableSwarmUI repo, or an outdated version

**Reply by rhd2330** [reply]
Is this the right version? SwarmUI v0.9.2.2 (2024-09-14 12:45:28)

**Reply by mcmonkey4eva** [reply]
that looks current yes

**Comment by mcmonkey4eva** [comment]
oh
`In the depth field, type 1.1; that is what I think I did, and everything on Swarm vanished under Loras.`
... yeah don't do that lol, set that back to `1` or `2` or something

**Comment by mcmonkey4eva** [comment]
I pushed a fix commit, hit Server -> Update&Restart, it will now properly display the page despite the invalid value, so you can set it back to something normal

**Reply by rhd2330** [reply]
Thank you so very much, that worked.

I am new to swarm, just exactly what values can be used, between what numbers?

**Reply by mcmonkey4eva** [reply]
just click the depth input and roll your scroll wheel or hit arrow keys up/down
